Borrow/Borrow.js

```diff
@@ -0,0 +1,56 @@
+const db = require("../db");
```

Comment:
Use single quote instead of a double quote.

Reply:
Done sir